.github/workflows/build-with-clang.yml

```diff
@@ -50,26 +52,25 @@ jobs:
       - name: Install mkl-service dependencies
         shell: bash -l {0}
         run: |
-          pip install numpy cython setuptools pytest pytest-cov
+          pip install cython setuptools pytest
```

Comment:
Alternatively there is an action might be used to install pip packages:
```yaml
        uses: BSFishy/pip-action@8f2d471d809dc20b6ada98c91910b6ae6243f318 # v1
        with:
          packages: |
            cython
            setuptools
            pytest
```
in case you find that useful.